LoadedData.html

```diff
@@ -11,30 +11,33 @@
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 </head>
 
-<nav class="navbar navbar-expand-lg navbar-light">
-  <a class="navbar-brand" href="">DATIM</a>
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo02" aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
-    <span class="navbar-toggler-icon"></span>
-  </button>
-
-
-  <div class="collapse navbar-collapse" id="navbarTogglerDemo02">
-    <ul class="navbar-nav">
-      <li class="nav-item">
-        <a class="nav-link" href="index.html">Home</a>
-      </li>
-      <li class="nav-item">
-        <a class="nav-link" href="LoadedData.html">Loaded Data</a>
-      </li>
-      <li class="nav-item">
-        <a class="nav-link" href="SpecialRequests.html">Special Requests</a>
-      </li>
-      <li class="nav-item">
-        <a class="nav-link" href="WhatsNext.html">What's Next to be Loaded?</a>
-      </li>
-    </ul>
-  </div>
-</nav>
+
+<div class="nav-container">
+  <nav class="navbar navbar-expand-lg navbar-light">
+    <a class="navbar-brand" href="" style="color:white;">DATIM</a>
```

Comment:
A few things here:
1. On the Loaded Data page, the "Plan Revision Statuses for NFS" link still has color contrast errors and needs that new class you created applied to it.
2. I see all of these navigation links in the header have `color: white;` applied to them. All of these links also have a class applied name `nav-link`. Could you just add a CSS rule for `nav-link` that applies this color instead of doing it manually per link? This way, if we ever need to make changes to the the navigation link color again, it'll hopefully be a one-line change.
3. In the ETL dialogs, the X button needs to be manually tested. Assuming I tested correctly, it seems like this X button fails when not hovered. Could you please fix this one, too?
4. If you zoom in far enough on these pages, a hamburger menu appears that lets you toggle the various links. Could you apply a similar coloring to these, too? This one seems to apply color a bit weirdly, so let me know if you need help figuring this part out.
![image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/72825162/bbb50d4a-9a4e-4398-815d-a49c039f3407)